src/ispc.h

```diff
@@ -407,6 +409,9 @@ class Target {
 
     /** Indicates whether the target has FP64 support. */
     bool m_hasFp64Support;
+
+    /** Indicates whether the target has uint32 -> float cvt support **/
+    bool m_hascvtu32Support;
```

Comment:
I don't like the name, as it's in line with other properties, which define **functionality**. This one is explicitly for warnings. I would name it `m_warnFtoU32IsExpensive`.

Reply:
Changed name to 'm_warnFtoU32IsExpensive'.